src/Kunstmaan/FixturesBundle/DependencyInjection/KunstmaanFixturesExtension.php

```diff
@@ -24,5 +26,11 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.yml');
+
+        if (!$container->hasDefinition('mime_types')) {
```

Comment:
Why is this added to the fixtures bundle?

Reply:
because 
src/Kunstmaan/FixturesBundle/Builder/MediaBuilder.php

also is refactored and uses the new mimetypes. If i didn't also do this check here it would trigger exceptions if you tried using the fixturesbundle without the media bundle